### PR TITLE
Lps 92429 DS for asset-list-service

### PR DESCRIPTION
**Pull request by lanceji92674957**

### CODE REVIEW

**Comment by lanceji92674957** [comment]
ci:test:relevant

File: modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/impl/AssetListEntryServiceImpl.java [hunk 1]
```diff
@@ -244,18 +253,22 @@ public AssetListEntry updateAssetListEntryTypeSettingsProperties(
...
-	@ServiceReference(type = CustomSQL.class)
+	@Reference(
+		policy = ReferencePolicy.DYNAMIC,
+		policyOption = ReferencePolicyOption.GREEDY,
+		target = "(model.class.name=com.liferay.asset.list.model.AssetListEntry)"
+	)
+	private volatile ModelResourcePermission<AssetListEntry>
+		_assetListEntryModelResourcePermission;
+
+	@Reference
```
Comment (Preston-Crary) [inline modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/impl/AssetListEntryServiceImpl.java]: For this one just use the target, we're ignoring the GREEDY and DYNAMIC now.

Comment (lanceji92674957) [reply modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/impl/AssetListEntryServiceImpl.java]: ok, I see

**Comment by lanceji92674957** [comment]
rebase and resent at https://github.com/Preston-Crary/liferay-portal/pull/721

**Pull request closed without merging**